core/src/Exceptions/CoreErrorCodes.cs

```diff
@@ -77,7 +77,9 @@ internal class CoreErrorCodes
         public const string UapCannotFindUpn = "uap_cannot_find_upn";
 
         public const string GetUserNameFailed = "get_user_name_failed";
-
+#if iOS
+        public const string GetTeamIdReturnedNull = "null_TeamId";
```

Comment:
Wouldn't the customer facing issue that the app cannot access the KeyChain for all the apps of the same publisher? What about `CannotAccessPublisherKeyChain` ?

Reply:
agreed. made the change and am also changing the response.

---
In reply to: [222515585](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1297#discussion_r222515585) [](ancestors = 222515585)